### ISSUE TITLE
Map not loading properly 

### ISSUE THREAD

**Issue by Zelheim** [open-sdg/open-sdg#2074]
**Please complete the following information**
Data repository URL: https://github.com/sdg-data-canada-odd-donnees/sdg-data-donnees-odd
Data or metadata issue: Data
Input format: CSV

**Describe the bug**
Map is not loading as expected for indicator 8.6.1:
https://sdg-data-canada-odd-donnees.github.io/sdg-odd-dev/8-6-1/
The map for all other indicators with maps are loading properly.

**Comment by christoph-LSN** [comment]
@Zelheim 
Looking at your cvs file there is no data in the GeoCode Column

"Year","Geography","Educational attainment level","Status","Sex","Age group","GeoCode","Value"
2015,,,,,,,13
2016,,,,,,,13
2017,,,,,,,12
2018,,,,,,,12
2019,,,,,,,11
2020,,,,,,,14
2021,,,,,,,14
2022,,,,,,,11
2015,"Canada","Total, all education levels","Total, not in employment, education or training (NEET)","Males","Total, 15 to 29 years",**GEOCODE**,13
2015,"Canada","Total, all education levels","Total, not in employment, education or training (NEET)","Females","Total, 15 to 29 years",,13


Additionally I saw there is a whitespace in your Agegroup column: , 15 to 29 (Between , and 15

**Comment by christoph-LSN** [comment]
@Zelheim inspecting the site shows this error:

sdg.js?v=20231122210745:164 Uncaught TypeError: Cannot read properties of undefined (reading 'getCurrentSeries')
    at Plugin.updateTitle (sdg.js?v=20231122210745:164:55)
    at HTMLLIElement.finalMapPreparation (sdg.js?v=20231122210745:698:16)
    at HTMLLIElement.dispatch (jquery-3.6.1.min.js:2:43064)
    at y.handle (jquery-3.6.1.min.js:2:41048)

not sure, but are there any data start values? Cannot find them in your meta-data files....

**Comment by brockfanning** [comment]
@Zelheim It seems like there is some aspect of the data for 8.6.1 that is causing javascript errors in Open SDG. I noticed that a lot of the rows of data in 8.6.1 have no Value. As an experiement, locally I removed any lines without a Value, and this allowed the map to load for me:
![image](https://github.com/open-sdg/open-sdg/assets/1319083/ac585b78-ed9b-49ac-b21f-fb728b0a46cd)

So one possible fix would be to remove rows without Values. If it is important to keep these rows, we would probably need to make an update to Open SDG to be able to handle this type of scenario without javascript errors.
